File: ARRAY cpp/lowest_number.cpp
#include<bits/stdc++.h> 
using namespace std;
int main(){
    int n;
    cin>>n;
    int array[n];

    for(int i=0;i<n;i++){
        int x;
        cin>>x;
        array[i]=x;
    }

    int min=array[0];
    for(int i=0;i<n;i++){
        if(array[i]<min){
            min=array[i];

        }
    }

    cout<<min<<endl;

}
////https://www.w3schools.com/dsa/dsa_data_arrays.php

File: ARRAY py/lowest_number.py
n=int(input("enter number of number->"))
array=[]
while n!=0:
    x=int(input("enter number->"))
    array.append(x)
    n-=1

mini=array[0]
i=0
for num in array:
    if num<mini:
        mini=num
    i+=1

print("the minimum number is",mini)

##https://www.w3schools.com/dsa/dsa_data_arrays.php